poglink/cogs/rates.py

```diff
@@ -74,43 +57,21 @@ async def get_current_rates(url):
             except Exception as e:
                 raise RatesFetchError(e) from e
 
-    async def send_embed(self, description, url, title=None):
+    async def send_embed(self, description, rates_dict):
+        url = rates_dict.get('url')
+        server_name = rates_dict.get('server_name')
+        server_color = rates_dict.get('color')
+        
         # generate embed
         logger.debug(f"Attempting to send embed. desc: {description}, url: {url}")
 
         # TODO: Add ability to accept custom URL/Title pairs, rather than parsing the DEFAULT_SERVER_INFO list for hardcoded options, then remove this if statement
-        if "atlas" in url:
-            if "pve" in url:
-                server_name = "PVE"
-                server_color = 0x63BCC3
-            else:
-                server_name = "PVP"
-                server_color = 0xA34C44
-        else:
-            match = re.match(
-                r"(?P<host>.*\/)?(?:(?P<platform>.*)\_(?P<game_mode>.*)\_)?dynamicconfig\.ini",
-                os.path.basename(urlparse(url).path),
-            )
-            if match:
-                server_match_dict = match.groupdict()
-
-            else:
-                logger.warning(f"Rates url was not recognized as any known type: {url}")
-                server_match_dict = {}
-
-            server_meta = self.DEFAULT_SERVER_INFO.get(
-                server_match_dict.get("game_mode")
-            )
-            logger.debug(f"Server meta: {server_meta}")
 
-            server_name = server_meta.get("short_name")
-            server_color = server_meta.get("color")
 
-        if title is None:
-            # generate dynamic timestamp (https://hammertime.djdavid98.art/)
-            ts = int(time.time() // 60 * 60)
-            ts_string = f"<t:{ts}:t>"
-            title = f"{server_name} server rates updated at {ts_string}"
+        # generate dynamic timestamp (https://hammertime.djdavid98.art/)
+        ts = int(time.time() // 60 * 60)
+        ts_string = f"<t:{ts}:t>"
+        title = f"{server_name} server rates updated at {ts_string}"
```

Comment:
removed the ability to add a custom title, we've never used it, I don't think its needed.